src/formatting.rs

```diff
@@ -245,7 +245,7 @@ enum Timer {
 
 impl Timer {
     fn start() -> Timer {
-        if cfg!(target_arch = "wasm32") {
+        if cfg!(all(target_arch = "wasm32", not(target_os = "wasi"))) {
```

Comment:
I wouldn't have been able to answer whether there was a potential for panic on Instant::now() with wasi, so good to know!